VIAFbotWD.py

```diff
@@ -408,4 +409,4 @@ def crawlLanguage(lang):
 
 crawlLanguage('en')
 
-print 'end'
\ No newline at end of file
+print 'end'
```

Comment:
Seems GitHub added this itself, but it probably doesn't hurt to have a newline at the end just in case anyway :)

Reply:
Thanks! @reosarevok